packages/api/lambdas/sqs-message-consumer.js

```diff
@@ -61,23 +87,26 @@ async function processQueues(event, dispatchFn) {
       visibilityTimeout,
       deleteProcessedMessage: false,
     });
-    log.info(`processing queue ${queueUrl}`);
 
-    const messageConsumerFn = dispatchFn.bind({ rulesForQueue });
+    log.info(`Processing queue ${queueUrl}`);
+    const messageConsumerFn = dispatchFn.bind({ queueUrl, rulesForQueue });
```

Comment:
I don't think we need to add `queueUrl` to this `bind` expression. `queueUrl` is already passed to the message consumer function:

https://github.com/nasa/cumulus/blob/master/packages/ingest/src/consumer.ts#L44

Reply:
[`5223038` (#2132)](https://github.com/nasa/cumulus/pull/2132/commits/522303829adf7b5c63fec589dced18ebcf084cf4)